ref/general/packaging-deploying.adoc

```diff
@@ -14,79 +14,69 @@
 = Package and deploy applications with Open Liberty
 
 You can choose among different options for packaging, containerizing, and deploying applications with Open Liberty.
-However, the simplest workflow for developing your cloud-native applications is to build applications with Maven or Gradle, containerize server packages with Docker, and deploy to the cloud with Kubernetes or OpenShift.
+However, the simplest workflow for developing your cloud-native applications is to build applications with Maven or Gradle, containerize with Docker, and deploy to the cloud with Kubernetes or OpenShift.
 
-== Package
-Before you containerize and deploy your application, package it with the Open Liberty runtime.
-There are different tools that you can use to package your application.
-Using Maven or Gradle to build and package your application is one of the easiest ways to get your application ready for cloud deployment.
-Maven and Gradle both work well as part of continuous integration and continuous deployment (CI/CD) pipelines.
-They start with the Open Liberty kernel and download specific feature versions that are listed as dependencies.
-
-=== Maven
+== Maven and Gradle build tools
 link:https://maven.apache.org/[Maven] is a build automation tool that makes managing your builds and dependencies easier.
 With Maven, you use a Project Object Model (POM) file written in XML to define information about your project, and Maven downloads and installs all dependencies for you.
 The link:https://github.com/OpenLiberty/ci.maven[Open Liberty Maven plug-in] helps manage your Open Liberty server and applications.
-
-After you enable the Maven plug-in, specify `war` in the `<packaging>` element in your `pom.xml` file.
-Then, run the `mvn package` command to package your application and Open Liberty into a WAR file.
-This command compiles, verifies, and builds the project.
-After you create this server package, you're ready to containerize and deploy it.
 To learn more about the Open Liberty Maven plug-in, check out this guide on link:/guides/maven-intro.html[Building a web application with Maven].
 
-=== Gradle
 Like Maven, link:https://gradle.org/[Gradle] is a build tool that provides dependency management for your projects.
 However, Gradle has it's own domain-specific language (DSL) that's based on Groovy.
```

Comment:
I wouldn't do (DSL) in this case since it isn't reference again. If you are doing a compare and contrast I would probably mention somewhere that the pom file is XML and gradle builds are essentially a program. It is a key difference.